pom.xml

```diff
@@ -45,7 +45,7 @@
   </scm>
   <issueManagement>
     <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/MSHARED/component/12326428</url>
+    <url>https://issues.apache.org/jira/browse/MSHARED-932?jql=project%20%3D%20MSHARED%20AND%20component%20%3D%20maven-dependency-analyzer</url>
```

Comment:
Could it point to JIRA without reference to specific issue, like MSHARED-932 here?

Reply:
Agree, this needs to be changed.

Reply:
What URL do you suggest? So far I have not been able to construct one that works and does not include an issue.

Reply:
> What URL do you suggest? So far I have not been able to construct one that works and does not include an issue.

Similar to one proposed in https://github.com/apache/maven-project-utils/pull/3#discussion_r453213245

Reply:
done